demos/ASP.Net Core API Gateway/Divergent.Sales.API.Host/Controllers/OrdersController.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Divergent.Sales.Data.Context;
+using System.Collections.Generic;
+using System.Data.Entity;
+using System.Linq;
+using System.Web.Http;
+
+namespace Divergent.Sales.API.Host.Controllers
+{
+    [RoutePrefix("api/orders")]
+    public class OrdersController : ApiController
+    {
+        [HttpGet]
+        public dynamic Get(int id)
```

Comment:
I checked and we return an anonymous type:

```
var order = db.Orders
    .Include(o => o.Items)
    .Where(o => o.Id == id)
    .Select(o => new
    {
        Number = o.Id,
        o.Id,
        ItemsCount = o.Items.Count
    })
    .SingleOrDefault();

return order;
```

in which case the return type can only be `dynamic` or `object`